data/human/south jobs.txt

```diff
@@ -15,7 +15,8 @@ mission "Pirate Occupation [0]"
 	name `Pirates occupying <system>`
 	minor
 	repeat
-	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> when done. Payment is <payment>.`
+	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> before <deadline>, to receive the payment of <payment>.`
```

Comment:
I don't know why I said `deadline`.
```suggestion
	description `A small band of pirates have been occupying <system>, and the local government has asked for help. Defeat the pirates in the system and land on <planet> before <date>, to receive the payment of <payment>.`
```

Reply:
It's fine, I can't remember which works and which doesn't.